vulns/gratient/PYSEC-0000-1368.yaml

```diff
@@ -0,0 +1,30 @@
+id: PYSEC-0000-1368
+modified: 2024-01-03T22:31:36.000000Z
+summary: gratient 0.5 contains credential harvesting code
+details: |
+  gratient is a user-facing library for generating color gradients of text.
+  Version 0.5 contained obfuscated, malicious code targeting
+  Windows platforms, harvesting information and credentials from the
+  user's system and sending them to a remote server.
+  Services may include Mullvad VPN and Telegram.
+affected:
+- package:
+    ecosystem: PyPI
+    name: gratient
+    purl: pkg:pypi/gratient
+  ranges:
+  - type: ECOSYSTEM
+    events:
+    - introduced: "0.5"
```

Comment:
Since the project itself isn't a threat anymore I think we can use only `versions` and leave the `range` out? I think that should work?

Reply:
Done!